apps/site/lib/site_web/router.ex

```diff
@@ -137,7 +137,13 @@ defmodule SiteWeb.Router do
     get(
       "/schedules/line_api/predictions_and_vehicles",
       ScheduleController.LineApi,
-      :predictions_and_vehicles
+      :realtime
+    )
+
+    get(
+      "/schedules/line_api/realtime",
+      ScheduleController.LineApi,
+      :realtime
```

Comment:
This fits on one line, I think (and maybe the above one does too, now that it's shorter)

Reply:
interesting...I'd have expected my Elixir formatting plugin to catch that

Reply:
`mix format` generally preserves your chosen line breaks even if it's possible to fit the statement on one line.